docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1730,7 +1731,7 @@ For example, the following P4~16~ objects involve complex types that need to be
 exposed in P4Runtime in order to support runtime operations on these objects.
 
 ~ Begin P4Example
-value_set<tuple<bit<16>, bit<8> > >(16) pvs_complex;
+value_set<tuple<bit<4>, bit<8> > >(16) pvs_complex;
```

Comment:
Why the change to 4 bits?

Reply:
This is really just a nit-pick change -- the field being compared against in the example is the IHL field of the IPv4 header, which is 4 bits wide in the standard.  This code snippet is short enough that it does not contain the explicit declaration of the IPv4 header type.  My OCD kicked in while reading it :-)

Reply:
@jafingerhut Got it, I was wondering.  And in this case,  OCD stands for "Occupational Capability Demonstration." :)

Reply:
Oh, and I was misremembering that it was the IPv4 IHL field, but it was another 4-bit field, the IPv4 version field.